scss/components/alert.scss

```diff
@@ -161,3 +161,8 @@ $block: #{$fd-namespace}-alert;
       }
     }
 }
+
+p {
+  margin-bottom: 0;
+  margin-top: 0;
+}
```

Comment:
Styles cannot be applied directly to elements (`<p>` in this case) as they can bleed into other components on the page.

Reply:
@greg-a-smith Thank you for the review, will create a class for it to prevent the bleeding.